usr/bin/nxdk-configure

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/sh
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
+
+export CC=nxdk-cc
+export PKG_CONFIG=nxdk-pkg-config
+
+# Wrapper around autoconf ./configure scripts
+./configure --host=i686-windows
```

Comment:
This really likes to find host libs. As mentioned here https://github.com/JayFoxRox/nxdk/pull/74#issuecomment-817182130 this will find X11 and OpenGL libs.

This is probably because the X11 search explicitly adds /usr/include etc.?

So this might need a better way to `chroot`

Reply:
Huh, I might have messed up my test. It only finds OpenGL because the test is broken: It only looks for "SDL_opengl.h".
It doesn't find X11 right now.